### PR TITLE
Add circular genome view storybook

### LINKED ISSUE #2462: Add storybook upload for embedded circular view
Current LGV docs are at `s3://jbrowse.org/storybook/lgv/`, so could use `s3://jbrowse.org/storybook/cgv/` or similar for this.

After docs are on the site, we can probably add the embedded circular view to https://jbrowse.org/jb2/download/#jbrowse-2-embedded-components

### PR DESCRIPTION
**Pull request by cmdcolin**
Follow up to #2508 to add circular view + adds storybook

Fixes #2462

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -108,3 +108,29 @@ jobs:
           pwd
           aws s3 sync --delete storybook-static s3://jbrowse.org/storybook/lgv/$(echo ${{github.ref}} | cut -d "/" -f3-)
         working-directory: products/jbrowse-react-linear-genome-view
+  storybook_cgv:
+    name: Build and deploy static Storybook site to AWS S3
```
Comment (rbuels) [inline .github/workflows/push.yml:112]: maybe rename to mention the embedded circular

**Review by rbuels — APPROVED** [review]
Looks good. Didn't @garrettjstevens make a start on something like this a while ago? What ended up happening to it?

**Comment by garrettjstevens** [comment]
> Looks good. Didn't @garrettjstevens make a start on something like this a while ago? What ended up happening to it?

This is based on the basic storybook I did, but I never added uploading or anything like this has.

**Comment by cmdcolin** [comment]
@garrettjstevens made the storybook for this already and exists on main, this PR essentially adds a github build step to make it so that this gets deployed to s3. it also adds a "getting started" mdx formatted page with some explainer text similar to the lgv storybook, and changes it so the "show code" shows something that users can see more easily in the storybook

**Pull request merged**